kepler_counterparts/kepler_counterparts.ipynb

```diff
@@ -0,0 +1,1509 @@
+{
```

Comment:
<p>I'm not wondering if this is kind of old school? Because it's the same output as if you just say print(tab) which I don't think was always the case. I'm actually taking it out for now, but am vacillating on whether or not to show it this way or just display the table the default way...</p>